src/server/api/lib/twilio.js

```diff
@@ -5,8 +5,6 @@ import { log } from '../../../lib'
 import { getLastMessage } from './message-sending'
 import faker from 'faker'
 
-const defaultService = process.env.DEFAULT_SERVICE
```

Comment:
This is the twilio file, so it should never be 'defaultService' -- it's 'twilio'.  The nexmo should probably have 'nexmo' hard-coded in the same way.  the DEFAULT_SERVICE environment variable is for the other parts of the app that need to decide which service to delegate to.